tests/src/test/java/alluxio/shell/command/ChgrpCommandTest.java

```diff
@@ -41,7 +41,8 @@ public void chgrpTest() throws IOException, AlluxioException {
   /**
    * Tests -R option for chgrp recursively.
    *
-   * @throws Exception
+   * @throws IOException if creating the file or getting the status of the filesystem fails
+   * @throws AlluxioException if getting the status of the filesystem fails
```

Comment:
We're trying to steer away from javadoc-ing exceptions in tests, or declaring explicit exceptions. The rationale is that unlike production code, tests are never supposed to throw an exception, and no other code calls test methods.